src/scenes/Languages/Edit/EditLanguage.tsx

```diff
@@ -33,6 +33,9 @@ export const EditLanguage = (props: EditLanguageProps) => {
               registryOfDialectsCode: language.registryOfDialectsCode.value,
               leastOfThese: language.leastOfThese.value,
               leastOfTheseReason: language.leastOfTheseReason.value,
+              isSignLanguage: language.isSignLanguage.value,
```

Comment:
Carson would like us to add `signLanguageCode` as well, if we can

Reply:
Added